packages/@sfx/search-driver-plugin/src/search-driver-plugin.ts

```diff
@@ -86,13 +93,17 @@ export default class SearchDriverPlugin implements Plugin {
    *
    * @param query the query to send.
    */
-  sendSearchApiRequest(query: BridgeQuery): Promise<Results> {
-    return this.core.search.search(query);
+  sendSearchApiRequest(query: string): Promise<Results> {
+    const fullQuery = { ...this.defaultSearchConfig, query };
+    return this.core.search.search(fullQuery);
```

Comment:
Going off the above ticket, this should prolly have some sort of callback to transform the results like the `sayt-driver` has, but again, prolly out of scope, correct?

Reply:
Good question. We aren't really using search results right now in general, with the exception of the `SaytDriver` calling the `Search` plugin directly for use within the Sayt web component. Let's leave this out of scope until we have a good reason to change it.